docs/Config.md

```diff
@@ -85,6 +85,7 @@ gui:
   skipRewordInEditorWarning: false # for skipping the confirmation before launching the reword editor
   border: 'rounded' # one of 'single' | 'double' | 'rounded' | 'hidden'
   animateExplosion: true # shows an explosion animation when nuking the working tree
+  portraitMode: false # always use a vertically stacked layout
```

Comment:
I'd suggest to make this a string with the possible values "auto", "never", "always".

Reply:
Sounds good. 

Where should the constants for these strings live? 

For instance, i can define them close to the UserConfig, but that maybe a bit awkward to import into package "gui". 

Could also just do the raw string matching in the gui package, without referencing any constant values for the strings. 

Lmk your opinion.

Reply:
In other cases where we use this pattern we don't define constants, we simply use string literals (e.g. for the "border" config right above). So it seems this is considered good enough. 😄